redaxo/src/addons/structure/plugins/content/pages/templates.php

```diff
@@ -26,12 +26,39 @@
         $error = rex_i18n::msg('csrf_token_invalid');
     } else {
         $del = rex_sql::factory();
-        $del->setQuery('SELECT ' . rex::getTablePrefix() . 'article.id,' . rex::getTablePrefix() . 'template.name FROM ' . rex::getTablePrefix() . 'article
+        $del->setQuery('SELECT ' . rex::getTablePrefix() . 'article.id, rex_article.clang_id, ' . rex::getTablePrefix() . 'template.name FROM ' . rex::getTablePrefix() . 'article
         LEFT JOIN ' . rex::getTablePrefix() . 'template ON ' . rex::getTablePrefix() . 'article.template_id=' . rex::getTablePrefix() . 'template.id
         WHERE ' . rex::getTablePrefix() . 'article.template_id="' . $template_id . '" LIMIT 0,10');
 
         if ($del->getRows() > 0 || rex_template::getDefaultId() == $template_id) {
-            $error = rex_i18n::msg('cant_delete_template_because_its_in_use', rex_i18n::msg('id') . ' = ' . $template_id);
+            $template_in_use_message = '';
+            $templatename = $del->getValue(rex::getTablePrefix(). 'template.name');
+            while ($del->hasNext()) {
+                $aid = $del->getValue(rex::getTablePrefix().'article.id');
+                $clang_id = $del->getValue(rex::getTablePrefix() . 'article.clang_id');
+                $OOArt = rex_article::get($aid, $clang_id);
+
+                $label = $OOArt->getName() . ' [' . $aid . ']';
+                if (rex_clang::count() > 1) {
+                    $label = '(' . rex_i18n::translate(rex_clang::get($clang_id)->getName()) . ') ' . $label;
+                }
```

Comment:
Bitte mit `else` und den `$OOA` nur dort erzeugen

Reply:
Oh sorry, ich sehe gerade dass sich dadurch das label geändert hat... es war ja auch im IF benutzt... dann macht meine Anmerkung keinen Sinn

Reply:
Änderung rückgängig gemacht, da sonst bei Mehrsprachigkeit der Artikelname und Id nicht mehr ausgegeben wurde.

Reply:
jep, danke. 👍

Reply:
Ist `$OOArt` überhaupt R5 oder nur aus Kompatibilitätsgründen noch drin?

Reply:
er verwendet die OOArticle klasse hier nicht, der variablen bezeichner `$OOArt` ist noch bissl R4 style das stimmt ;)

Reply:
Ist nur der Variablenname, und diesen Part hat er sich vermutlich kopiert (von der is_in_use-Geschichte aus dem Medienpool eventuell).
In der Tat wäre es aber nicht ganz verkehrt, die Variable `$article` zu nennen.